src/Model/GroundWaterTransport/gwt.f90

```diff
@@ -337,6 +339,29 @@ subroutine gwt_rp(this)
     return
   end subroutine gwt_rp
 
+  !> @brief GWT Model time step size
+  !!
+  !! Calculate the maximum allowable time step size subject to time-step
+  !! constraints.  If adaptive time steps are used, then the time step used
+  !! will be no larger than dtmax calculated here.
+  !<
+  subroutine gwt_dt(this)
+    use TdisModule, only: kstp, kper
+    use AdaptiveTimeStepModule, only: ats_submit_delt
```

Comment:
I'm torn on this one.  I think the style guide is right to say that we prefer module level imports rather than routine imports, but if the imports are only used in a single routine, it seems reasonable to import them in the routine directly.  Think we may need further discussion on this.